### PR TITLE
doc(support svg sprite loader)

### PR DESCRIPTION
**Pull request by Athira001**
https://github.com/quintype/ace-planning/issues/454

Please include the summary of the change made. Please include the required context here. Please include the issue reference for the fix and the dependencies reference for the change.

Fixes # (issue-reference)

Dependencies # (dependency-issue-reference)

Documentation # (link to the corresponding documentation changes)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test Case A
- [ ] Test Case B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ags1773 — APPROVED** [review]

File: tutorial/support-svg-sprite.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+**Scalability and resolution** - This makes it possible to zoom in and out of SVG images without losing any quality and gives SVG the ability to scale infinitely
+
+**Minimal file size** - The file sizes of SVG images are usually small compared to other file formats and they are easily compressible
+
+**High performance and speed** - Because of the small size nature of SVG images, it becomes very easy and fast for browsers to render them. 
+
+**DOM-like and style-able** - SVG images are like code, so this means it can be navigated like a DOM element and also styled, although some properties have different names eg(you might want to use fill instead of color). It can also be styled with CSS.
+
+**Editable** - Since SVGs are DOM-like, SVG’s can be created and edited
+
```
Comment (nandap4790) [inline tutorial/support-svg-sprite.md:25]: Till here it is not needed

File: tutorial/support-svg-sprite.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+## What is SVG's in JS?
+
+Like HTML, SVGs are represented using the [Document Object Model](https://en.wikipedia.org/wiki/Document_Object_Model "Document Object Model") (DOM) and so can be manipulated with Javascript relatively easily.
```
Comment (nandap4790) [inline tutorial/support-svg-sprite.md:28]: ?

File: tutorial/support-svg-sprite.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+SVG's should be made external via webpack / plugins to load them lazy via a network call or DOM reference. Thus we are using `SVG sprite loader` (Webpack loader for creating SVG sprites.) 
+
+## Why SVG sprite loader?
+SVG Sprites are rendered and injected in pages automatically, you just refer to images via `<svg><use xlink:href="#id"></use></svg>`. Also it can render sprites on server or in browser manually.  
+
+## Set up [SVG sprite loader](https://www.npmjs.com/package/svg-sprite-loader "SVG sprite loader") for your app
+
+There are few steps you need to follow to set up SVG sprite loader:
+
+1. The first thing you need to install the package `npm install svg-sprite-loader -D`
```
Comment (nandap4790) [inline tutorial/support-svg-sprite.md:45]: grammarly

File: tutorial/support-svg-sprite.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+webpackConfig.plugins.push(svgPlugin());
+
+module.exports = {
+  ...webpackConfig
+};
+
+```
+Once you compile(build) the webpack, `sprite.svg` will be generated and available in the public folder.
+
+3 .  Runtime configuration
```
Comment (nandap4790) [inline tutorial/support-svg-sprite.md:87]: indentation

File: tutorial/support-svg-sprite.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+  ...webpackConfig
+};
+
+```
+Once you compile(build) the webpack, `sprite.svg` will be generated and available in the public folder.
+
+3 .  Runtime configuration 
+When you require an image, the loader transforms it to SVG <symbol>, adds it to the special sprite storage and returns class instance that represents symbol. It contains id, viewBox and content (id, viewBox and url in extract mode) fields and can later be used for referencing the sprite image.
+
+ - Add all the svg icons inside `/assets/icons/` folder.
```
Comment (nandap4790) [inline tutorial/support-svg-sprite.md:90]: This comes without any heading

**Review by nandap4790 — CHANGES_REQUESTED** [review]

File: tutorial/support-svg-sprite.md (hunk 5 above)
Comment (Athira001) [reply tutorial/support-svg-sprite.md:90]: the first point in   Runtime configuration

File: tutorial/support-svg-sprite.md (hunk 4 above)
Comment (Athira001) [reply tutorial/support-svg-sprite.md:87]: done

File: tutorial/support-svg-sprite.md (hunk 3 above)
Comment (Athira001) [reply tutorial/support-svg-sprite.md:45]: Already done

File: tutorial/support-svg-sprite.md (hunk 1 above)
Comment (Athira001) [reply tutorial/support-svg-sprite.md:25]: Deleted

File: tutorial/support-svg-sprite.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+In this chapter, we are going to discuss how to use `SVGs in JS` without causing an impact on performance.
+
+## What is SVG's in JS?
+
+Like HTML, SVGs are represented using the [Document Object Model](https://en.wikipedia.org/wiki/Document_Object_Model "Document Object Model") (DOM) and so can be manipulated with Javascript relatively easily. 
+### Problem
+There are SVG's embedded in JS for icons / logos / etc. These cause an impact on performance because:
+- The size of the JS increases with additions and entail cost over the network
+- Render blocking
+- Downloaded JS is parsed affecting FCP & FMP.
```
Comment (nandap4790) [inline tutorial/support-svg-sprite.md:20]: the full stops are not consistent

Comment (nandap4790) [reply tutorial/support-svg-sprite.md:20]: check everywhere

File: tutorial/support-svg-sprite.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+  test: /\.svg$/,
+  loader: "svg-sprite-loader",
+  options: {
+    extract: true,
+    publicPath: "/",
+    symbolId: filePath => {
+      return path
+        .basename(filePath)
+        .replace(".svg", "")
+        .toLowerCase(); //
```
Comment (nandap4790) [inline tutorial/support-svg-sprite.md:52]: what is //

**Review by nandap4790 — APPROVED** [review]

**Pull request merged**